plugins/trace_api_plugin/include/eosio/trace_api/store_provider.hpp

```diff
@@ -255,6 +310,14 @@ namespace eosio::trace_api {
        */
       std::optional<data_log_entry> read_data_log( uint32_t block_height, uint64_t offset ) {
          const uint32_t slice_number = _slice_directory.slice_number(block_height);
+
+         // Prefer a compressed trace if one exists
```

Comment:
Shouldn't you prefer an uncompressed over a compressed?

Reply:
I could see that making sense, will change.